docs/.env

```diff
@@ -0,0 +1 @@
+FEEDBACK_URL=https://hgvi836wi8.execute-api.us-east-1.amazonaws.com
```

Comment:
It could be an environment variable set by Netlify.

@oliviertassinari @mbrookes Is there a reason that forces to use `docs/.env` instead of an envvar? I didn't find any comment considering this option in https://github.com/mui/material-ui/pull/22885

Reply:
I don't recall any specific reason, so probably just an implementation problem. I do remember that I found other issues (unrelated to this), but the record of those is lost in a Slack comment (I should really have opened a GitHub issue), and I haven't looked at it since. It certainly isn't perfect, so feel free to improve it in whatever way you see fit.

Reply:
I'll move it to Netlify environment then